markup.go

```diff
@@ -113,6 +114,14 @@ func (l *EventListener) PreventDefault() *EventListener {
 	return l
 }
 
+// StopPropagation prevents further propagation of the current event in the capturing and bubbling phases.
```

Comment:
I've been using 100 chars as a guide, but I'm perfectly happy with 80. Obviously there's times like [this](https://github.com/kego/ke/blob/2a9338538fc5913b2bb3c986aad826bbe02ea152/system/node/node.go#L24) where breaking a comment would cause gofmt to break the nice indenting.